ide/team.commons/src/org/netbeans/modules/team/spi/TeamAccessor.java

```diff
@@ -35,7 +35,7 @@
 public abstract class TeamAccessor {
 
     public static final String PROP_LOGIN = "team.login.changed";               // NOI18N
-    public static final String PROP_PROJETCS_CHANGED = "team.projects.changed"; // NOI18N
+    public static final String PROP_PROJECTS_CHANGED = "team.projects.changed"; // NOI18N
```

Comment:
You're changing a public NetBeans API/SPI without prior discussion.

Reply:
fixed in 0a41da3a
mark previos name as deprecated

Reply:
Marking things as deprecated is not enough when you're changing an API. To change a public API/SPI you have to:

* Ask in the mailing list if the change is acceptable or not.
* Ask in the mailing list for instructions on how to change a public API (marking things as "deprecated" is not good enough).

Thanks!